doc/release-notes/7400-opendp-download.md

```diff
@@ -0,0 +1,10 @@
+Auxiliary Files can now be downloaded from the web interface.
+
+- Aux files uploaded as type=DP appear under "Differentially Private Statistics" under file level download. The rest (type=OTHER) appear under "Other Auxiliary Files".
```

Comment:
it's now more than just type=OTHER

Reply:
Right, and type=OTHER isn't a thing anymore, so I removed this reference to it in 20b438201.